util/testutil/integration/dockerd.go

```diff
@@ -25,6 +25,11 @@ func InitDockerdWorker() {
 		unsupported: []string{
 			FeatureCacheExport,
 			FeatureCacheImport,
+			FeatureCacheBackendAzblob,
+			FeatureCacheBackendGha,
+			FeatureCacheBackendLocal,
+			FeatureCacheBackendRegistry,
+			FeatureCacheBackendS3,
 			FeatureDirectPush,
```

Comment:
Could you add feature validation for Merge and Diff as well for #45153. In a separate commit.

Reply:
https://github.com/moby/buildkit/pull/3711/commits/94d888325b0aedfd7b37f0ea03fbadb638394a05